priv/repo/migrations/20190404190924_create_users.exs

```diff
@@ -3,8 +3,8 @@ defmodule Renaissance.Repo.Migrations.CreateUsers do
 
   def change do
     create table(:users) do
-      add :email, :string
-      add :password_hash, :string
+      add :email, :string, null: false
```

Comment:
This migration is already checked in, right? You shouldn't modify old migrations instead make new ones. If this had already been deployed, it wouldn't actually take effect

Reply:
Also it doesn't convey the history of the changes

Reply:
Too true–will resist this cheat going forward :flushed: